src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        Colors figureColor = Colors.values()[index];
+        return figureColor.toString();
+    }
+
+    public String getConstantColor() {
+        Colors figureColor = Colors.values()[5];
```

Comment:
no need for this method, it can just be a public constant if you wish. [5] is unclear, it's a magic number. Use value directly